runhouse/main.py

```diff
@@ -610,138 +610,140 @@ def get_clusters_from_den():
-            not in clusters_in_den_names
-        ]
 
-    running_clusters = []
-    not_running_clusters = []
+def _add_clusters_to_output_table(
+    table: Table, clusters: list[Dict], filters_requested: dict = None
+):
+    for rh_cluster in clusters:
+        last_active_at = rh_cluster.get("Last Active (UTC)")
+        last_active_at_no_offset = str(last_active_at).split("+")[
+            0
+        ]  # The split is required to remove the offset (according to UTC)
+        rh_cluster["Last Active (UTC)"] = last_active_at_no_offset
+        rh_cluster["Status"] = StatusColors.get_status_color(rh_cluster.get("Status"))
 
-    for den_cluster in clusters_in_den:
-        # get just name, not full rns address. reset is used so the name will be printed all in white.
-        cluster_name = f'[reset]{den_cluster.get("name").split("/")[-1]}'
-        cluster_type = den_cluster.get("data").get("resource_subtype")
-        cluster_status = (
-            den_cluster.get("status") if den_cluster.get("status") else None
-        )
+        table = _add_cluster_as_table_row(table, rh_cluster)
 
-        # currently relying on status pings to den as a sign of cluster activity.
-        # The split is required to remove milliseconds and the offset (according to UTC) from the timestamp.
-        # (status_last_checked is in the following format: YYYY-MM-DD HH:MM:SS.ssssss±HH:MM)
 
-        last_active_at = den_cluster.get("status_last_checked")
-        last_active_at = (
-            datetime.datetime.fromisoformat(last_active_at.split(".")[0])
-            if isinstance(last_active_at, str)
-            else None
+def _print_msg_live_sky_clusters(sky_clusters: int):
+    if sky_clusters > 0:
+        console.print(
+            f"There are {sky_clusters} live cluster(s) that are not saved in Den. For more information, please run [bold italic]sky status -r[/bold italic]."
         )
-        last_active_at = (
-            last_active_at.replace(tzinfo=datetime.timezone.utc)
-            if last_active_at
-            else None
+
+
+def supported_time_units():
+    return ", ".join(list(TIME_UNITS.keys()))
+
+
+@cluster_app.command("list")
+def cluster_list(
+    show_all: bool = typer.Option(
+        False,
+        "--a",
+        "--all",
+        help="Get all den clusters. Up to 50 most recently active clusters will be displayed.",
+    ),
+    since: Optional[str] = typer.Option(
+        None,
+        "--since",
+        help="Time duration to filter on. Minimum time to filter on is 1 minute. If the provided value will be less "
+        "than that, the filter will be set to 1 minute by default. You may filter by seconds (s), "
+        "minutes (m), hours (h) or days (s). Examples: 30s, 15m, 2h, 3d.",
+    ),
+    cluster_status: Optional[str] = typer.Option(
+        None,
+        "--status",
+        help="Cluster status to filter on. Supported filter values: running, terminated, runhouse_daemon_down.",
```

Comment:
thoughts on changing the input option to `down` instead to make it easier for the CLI command? can still internally use the `runhouse_daemon_down` value